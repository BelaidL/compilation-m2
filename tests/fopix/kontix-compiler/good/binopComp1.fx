../../interpreter/good/binopComp1.fx