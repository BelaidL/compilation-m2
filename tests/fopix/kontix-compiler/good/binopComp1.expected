../../interpreter/good/binopComp1.expected